tools/tpch-tools/run-tpch-queries.sh

```diff
@@ -0,0 +1,117 @@
+#!/usr/bin/env bash
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+##############################################################
+# This script is used to create TPC-H tables
+##############################################################
+
+set -eo pipefail
+
+ROOT=$(dirname "$0")
+ROOT=$(
+    cd "$ROOT"
+    pwd
+)
+
+CURDIR=${ROOT}
+QUERIES_DIR=$CURDIR/queries
+
+usage() {
+    echo "
+This script is used to run TPC-H 22queries, 
+will use mysql client to connect Doris server which parameter is specified in doris-cluster.conf file.
+Usage: $0 
+  "
+    exit 1
+}
+
+OPTS=$(getopt \
+    -n $0 \
+    -o '' \
+    -- "$@")
+
+eval set -- "$OPTS"
+HELP=0
+
+if [ $# == 0 ]; then
+    usage
+fi
+
+while true; do
+    case "$1" in
+    -h)
+        HELP=1
+        shift
+        ;;
+    --)
+        shift
+        break
+        ;;
+    *)
+        echo "Internal error"
+        exit 1
+        ;;
+    esac
+done
+
+if [[ ${HELP} -eq 1 ]]; then
+    usage
+    exit
+fi
+
+check_prerequest() {
+    local CMD=$1
+    local NAME=$2
+    if ! $CMD; then
+        echo "$NAME is missing. This script depends on mysql to create tables in Doris."
+        exit 1
+    fi
+}
+
+check_prerequest "mysql --version" "mysql"
+
+source $CURDIR/doris-cluster.conf
+
+echo "FE_HOST: $FE_HOST"
+echo "FE_QUERY_PORT: $FE_QUERY_PORT"
+echo "USER: $USER"
+echo "PASSWORD: $PASSWORD"
+echo "DB: $DB"
+
+pre_set() {
+    echo $@
+    mysql -h$FE_HOST -u$USER -P$FE_QUERY_PORT -D$DB -e "$@"
+}
+
+pre_set "set global enable_vectorized_engine=1;"
```

Comment:
thanks your pr！

I have some questions
1.Can we specify the size of the generated data？
2.Can we specify the params？for example `enable_vectorized_engine=false`

following is my opinion：
1.we can user add more params such as `runtime filter、join reorder and so on`
2.can we simplify the steps？i think five steps are too complicated，can we merge step3 and step4？

Reply:
> thanks your pr！
> 
> I have some questions 1.Can we specify the size of the generated data？ 2.Can we specify the params？for example `enable_vectorized_engine=false`
> 
> following is my opinion： 1.we can user add more params such as `runtime filter、join reorder and so on` 2.can we simplify the steps？i think five steps are too complicated，can we merge step3 and step4？

Further, maybe we can unify a benchmark framework and let it support test sets such as ssb/tpch/tpcds...

Reply:
> Further, maybe we can unify a benchmark framework and let it support test sets such as ssb/tpch/tpcds...

good proposal